src/otx/config/data/mmdet.yaml

```diff
@@ -0,0 +1,14 @@
+defaults:
+  - default
```

Comment:
Instead of referring to the default config that has only two lines, can't we just add those parameters here?